pages/newsletter/index.tsx

```diff
@@ -0,0 +1,190 @@
+import styled from "styled-components";
+import { css, StyledSystemProps } from "components/system";
+import Head from "components/Head";
+import Link from "components/Link";
+import Box from "components/Box";
+import Flex from "components/Flex";
+import Image from "components/Image";
+import { NewsletterEmailSubscribe } from "components/Newsletter/NewsletterEmailSubscribe";
+
+import webBackground from "./assets/newsletter_web.png";
+import mobBackground from "./assets/newsletter_mob.png";
+import engineers from "./assets/engineers.png";
+import whiteLogo from "./assets/white_logo@1x.png";
+
+const Newsletter = () => {
+  return (
+    <>
+      <Head title="Newsletter" description="Subsribe to our Newsletter!" />
+      <StyledBackground
+        backgroundImage={[`url(${mobBackground})`, `url(${webBackground})`]}
+        height="100vh"
+      >
+        <Box mt={[9, 7]} mb={[5, 5]}>
+          <Box as="h1" m={0}>
+            <Link href="/">
+              <Flex
+                justifyContent="center"
+                alignItems="top"
+                height={["24px", "auto"]}
+                width={["120px", "auto"]}
+                css={css({
+                  "&:hover, &:active, &:focus": {
+                    opacity: 0.8,
+                  },
+                })}
+              >
+                <Image
+                  src={whiteLogo}
+                  alt="network infrastructure"
+                  width="100%"
+                />
+              </Flex>
+            </Link>
+          </Box>
+        </Box>
+        <Card>
+          <AccessPlane order={[2, 1]}>
+            <StyledBackground
+              backgroundImage={`url(${engineers})`}
+              backgroundSize="100%"
+              width={[282, 534]}
+              height={[158, 300]}
+              backgroundRepeat="no-repeat"
+              ml={[5, -3]}
+              mt={[4, 8]}
+            />
+            <Box>
+              <Box
+                fontSize={["header-4", "header-3"]}
+                lineHeight={["22px", "24px"]}
```

Comment:
```suggestion
                lineHeight={["22px", "md"]}
```

Nit: Probably ok just to make both `md, but need to check`.